src/Protocol/Authenticator.cpp

```diff
@@ -1,19 +1,18 @@
 
 #include "Globals.h"  // NOTE: MSVC stupidness requires this to be the same across all modules
 
-#include "Authenticator.h"
-#include "MojangAPI.h"
-#include "../Root.h"
-#include "../Server.h"
-#include "../ClientHandle.h"
-#include "../UUID.h"
-
-#include "../IniFile.h"
-#include "../JsonUtils.h"
+#include "Protocol/Authenticator.h"
+#include "Protocol/MojangAPI.h"
+#include "Root.h"
+#include "Server.h"
+#include "ClientHandle.h"
+#include "UUID.h"
+#include "HTTP/UrlParser.h"
+
+#include "IniFile.h"
+#include "JsonUtils.h"
```

Comment:
What's with the changes in the includes? Keep them relative, as they were.

Reply:
see #4901

Reply:
Oh well, okay for now. But I will get that decision reversed! :D